r/tools/autobrew

```diff
@@ -62,3 +62,5 @@ unset HOMEBREW_NO_AUTO_UPDATE
 
 # Hack to work around segfault on CRAN server
 # rm -f tests/testthat.R
+
+exit 1
```

Comment:
The log is correctly displayed.

Reply:
Yes, I didn’t trigger a failure in the right place, will try again